priv/www/api/index.html

```diff
@@ -437,6 +485,8 @@ <h2>Reference</h2>
         <td></td>
         <td>X</td>
         <td class="path">/api/exchanges/<i>vhost</i>/<i>name</i>/publish</td>
+        <td>management</td>
+        <td>User added to vhost & write permission</td>
```

Comment:
This should be clear as to what the `write` permission is on: the target exchange.